rollup.config.js

```diff
@@ -63,12 +63,17 @@ if (devMode || demoMode) {
 }
 
 if (devMode) {
-  // Create a server for dev mode
-  plugins.push(serve({ contentBase: 'dist', port: 10001 }));
-  // Allow to livereload on any update
+  let serverPort = 10002;
```

Comment:
this is not exactly what we want: the port is now `10002` for start and `10001`
If I run npm start, I cannot run the e2e tests as they the `10002` port is already in use